FIPS tests are broken in CI

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

testsuite

### Describe the bug

The combination of two merges have broken the FIPS tests: af717f70456b37422d27ab7b48cc357d78a3f92e (issue https://github.com/keycloak/keycloak/issues/48412)  and f4a783eaaf63a158271f7d9a9b53aaedd6fa7328 (issue https://github.com/keycloak/keycloak/issues/49139). The problem is that in the RHEL container the default `cacerts` file is a JKS store, that is not allowed by FIPS. When using the PEM truststore the default cacerts is added (JKS) to the default truststore. The previous test used direct pkcs12 or bcfps file which is OK as no addition is done (using the mTLS store). In my change this worked because the CryptoIntegration was not already started (now with the other change the crypto is already executed and BC is managing the default truststore setup). In the other change worked because the old FIPS testsuite is always using the mTLS store.

Part of the exception:

```
java.lang.RuntimeException: Failed to start quarkus
	at io.quarkus.runner.ApplicationImpl.<clinit>(Unknown Source)
	at java.base/jdk.internal.misc.Unsafe.allocateInstance(Native Method)
	at java.base/java.lang.invoke.DirectMethodHandle.allocateInstance(DirectMethodHandle.java:500)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:78)
	at org.keycloak.quarkus.runtime.KeycloakMain.start(KeycloakMain.java:145)
	at org.keycloak.quarkus.runtime.cli.Picocli.start(Picocli.java:913)
	at org.keycloak.quarkus.runtime.cli.command.AbstractAutoBuildCommand.runCommand(AbstractAutoBuildCommand.java:115)
	at org.keycloak.quarkus.runtime.cli.command.AbstractCommand.lambda$call$0(AbstractCommand.java:66)
	at java.base/java.util.Optional.orElseGet(Optional.java:364)
	at org.keycloak.quarkus.runtime.cli.command.AbstractCommand.call(AbstractCommand.java:65)
	at org.keycloak.quarkus.runtime.cli.command.AbstractCommand.call(AbstractCommand.java:34)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2031)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2469)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2423)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2425)
	at picocli.CommandLine.execute(CommandLine.java:2174)
	at org.keycloak.quarkus.runtime.cli.Picocli.execute(Picocli.java:197)
	at org.keycloak.quarkus.runtime.cli.Picocli.parseAndRun(Picocli.java:186)
	at org.keycloak.quarkus.runtime.KeycloakMain.main(KeycloakMain.java:117)
	at org.keycloak.quarkus.runtime.KeycloakMain.main(KeycloakMain.java:92)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:86)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.main(QuarkusEntryPoint.java:37)
Caused by: java.lang.RuntimeException: Failed to initialize truststore: /usr/lib/jvm/java-25-openjdk/lib/security/cacerts, type: PKCS12
	at org.keycloak.truststore.TruststoreBuilder.loadStore(TruststoreBuilder.java:226)
	at org.keycloak.truststore.TruststoreBuilder.includeDefaultTruststore(TruststoreBuilder.java:204)
	at org.keycloak.truststore.TruststoreBuilder.createMergedTruststore(TruststoreBuilder.java:122)
	at org.keycloak.truststore.TruststoreBuilder.setSystemTruststore(TruststoreBuilder.java:59)
	at org.keycloak.quarkus.runtime.KeycloakRecorder.configureTruststore(KeycloakRecorder.java:152)
	at io.quarkus.runner.recorded.KeycloakProcessor$configureTruststore2112467525.deploy_0(Unknown Source)
	at io.quarkus.runner.recorded.KeycloakProcessor$configureTruststore2112467525.deploy(Unknown Source)
	... 25 more
Caused by: java.io.IOException: stream does not represent a PKCS12 key store
	at org.bouncycastle.jcajce.provider.ProvPKCS12$PKCS12KeyStoreSpi.engineLoad(Unknown Source)
	at java.base/java.security.KeyStore.load(KeyStore.java:1502)
	at org.keycloak.common.util.KeystoreUtil.loadKeyStore(KeystoreUtil.java:114)
	at org.keycloak.truststore.TruststoreBuilder.loadStore(TruststoreBuilder.java:223)
	... 31 more
```

I'm changing the FIPS suites to use the mTLS store as the old testsuite was doing.

### Version

999.0.0-SNAPSHOT

### Regression

- [ ] The issue is a regression

### Expected behavior

FIPS CI should work OK.

### Actual behavior

FIPS CI is failing. For example: https://github.com/keycloak/keycloak/actions/runs/27007552476/job/79714650042?pr=49622

### How to Reproduce?

It's a CI issue.

### Anything else?

_No response_

--- a/tests/base/src/test/java/org/keycloak/tests/suites/FipsNonStrictTestSuite.java
+++ b/tests/base/src/test/java/org/keycloak/tests/suites/FipsNonStrictTestSuite.java
@@ -63,7 +63,7 @@ public class FipsNonStrictTestSuite {
                     .tlsEnabled(true)
                     .mTlsEnabled(true)
                     .keystoreFormat(KeystoreUtil.KeystoreFormat.BCFKS)
-                    .stores("keycloak.bcfks", "keycloak-truststore.pem", "client.bcfks", "keycloak-truststore.bcfks");
+                    .stores("keycloak.bcfks", "keycloak-truststore.bcfks", "client.bcfks", "keycloak-truststore.bcfks");
         }
     }
 }

--- a/tests/base/src/test/java/org/keycloak/tests/suites/FipsStrictTestSuite.java
+++ b/tests/base/src/test/java/org/keycloak/tests/suites/FipsStrictTestSuite.java
@@ -66,7 +66,7 @@ public class FipsStrictTestSuite {
                     .tlsEnabled(true)
                     .mTlsEnabled(true)
                     .keystoreFormat(KeystoreUtil.KeystoreFormat.BCFKS)
-                    .stores("keycloak.bcfks", "keycloak-truststore.pem", "client.bcfks", "keycloak-truststore.bcfks");
+                    .stores("keycloak.bcfks", "keycloak-truststore.bcfks", "client.bcfks", "keycloak-truststore.bcfks");
         }
     }
 }

